### PR TITLE
Update: 7S2P.Effie.CN version 2.1.19

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
### Result: Installation Successful
|             | Manifest                 | Add/Remove Programs                   |
| ----------- | ------------------------ | ------------------------------------- |
| Name        | Effie | Effie 2.1.19    |
| Version     | 2.1.19     |  |
| Publisher   | Shanghai 7S2P Technology Co., Ltd.   | 7S2P Inc.      |
| ProductCode |           | Effie_is1    |
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/71669)

### CODE REVIEW

File: manifests/7/7S2P/Effie/CN/2.1.19/7S2P.Effie.CN.locale.zh-CN.yaml [hunk 1]
```diff
@@ -30,8 +30,7 @@ Tags:
 - 编辑器
 - 脑图
 # Agreements: 
-ReleaseNotes: |-
-  - 支持 WebP 图片。
+# ReleaseNotes:
```
Comment (Trenly) [inline manifests/7/7S2P/Effie/CN/2.1.19/7S2P.Effie.CN.locale.zh-CN.yaml:33]:
```suggestion
ReleaseNotes: |-
  - 支持 WebP 图片。
```

File: manifests/7/7S2P/Effie/CN/2.1.19/7S2P.Effie.CN.locale.zh-CN.yaml [hunk 2]
```diff
@@ -1,8 +1,8 @@
-# Created with YamlCreate.ps1 v2.1.3 $debug=QUSU.7-2-6
+# Created with YamlCreate.ps1 v2.1.3 using InputObject 🤖 =QUSU.7-2-6
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.2.0.schema.json
 
 PackageIdentifier: 7S2P.Effie.CN
-PackageVersion: 2.1.19 # null
+PackageVersion: 2.1.19
 PackageLocale: zh-CN
 Publisher: Shanghai 7S2P Technology Co., Ltd.
```
Comment (Trenly) [inline manifests/7/7S2P/Effie/CN/2.1.19/7S2P.Effie.CN.locale.zh-CN.yaml:7]:
```suggestion
Publisher: 7S2P Inc.
```

File: manifests/7/7S2P/Effie/CN/2.1.19/7S2P.Effie.CN.locale.en-US.yaml [hunk 3]
```diff
@@ -1,8 +1,8 @@
-# Created with YamlCreate.ps1 v2.1.3 $debug=QUSU.7-2-6
+# Created with YamlCreate.ps1 v2.1.3 using InputObject 🤖 =QUSU.7-2-6
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
 
 PackageIdentifier: 7S2P.Effie.CN
-PackageVersion: 2.1.19 # null
+PackageVersion: 2.1.19
 PackageLocale: en-US
 Publisher: Shanghai 7S2P Technology Co., Ltd.
```
Comment (Trenly) [inline manifests/7/7S2P/Effie/CN/2.1.19/7S2P.Effie.CN.locale.en-US.yaml:7]:
```suggestion
Publisher: 7S2P Inc.
```

**Review by Trenly — CHANGES_REQUESTED** [review]
I would suggest adding ARP entries to all versions of this package to help with version mapping, since the version number is only in the name

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**